composer.json

```diff
@@ -30,7 +30,8 @@
 	},
 	"require-dev": {
 		"yoast/yoastcs": "dev-master",
-		"codeclimate/php-test-reporter": "dev-master"
+		"codeclimate/php-test-reporter": "dev-master",
+		"wp-coding-standards/wpcs": "0.6.0"
```

Comment:
Why here? This is dependency of YoastCS already. If we need to bump version should update it there, then update lock file downstream (here).